src/main/java/edu/ucsb/mapache/services/GoogleSearchService.java

```diff
@@ -161,6 +170,30 @@ public void saveToSearchRepository(String searchQuery) {
         searchRepository.save(s);
     }
 
+    public void saveToUserSearchRepository(String searchQuery,String authorization){
+	    UserSearch s;
+	    
+	     DecodedJWT jwt = getJWT(authorization);
+        
+         Map<String, Object> customClaims = jwt.getClaim(namespace).asMap();
+        
+        s=new UserSearch();
+	    
+        String firstName = (String) customClaims.get("given_name");
+        String lastName = (String) customClaims.get("family_name");
+	    String userid=firstName.concat(lastName);
+        s.setSearchTerm(searchQuery);
+	
+        s.setUserID(userid);
+	    String timestamp = new SimpleDateFormat("yyyy-MM-dd HH:mm:ss.SSS Z").format(Calendar.getInstance().getTime());
+        s.setTimestamp(timestamp);
+
+	    usersearchRepository.save(s);
+
+    }
```

Comment:
I wonder whether you might make the indentation more consistent.

Reply:
Code Formatting in VS Code:
On Windows Shift + Alt + F
On Mac Shift + Option + F
On Linux Ctrl + Shift + I

Reply:
Change the indentation.